### PR TITLE
[StyleCop] Fix all the warnings in Number/Spanish Extractor and Parser files

### PR DESCRIPTION
**Pull request by Aliandi**
Fixes
- Reorder properties
- Add trailing comma
- Add spacing when needed
- Remove trailing whitespace

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.Number/Spanish/Extractors/NumberExtractor.cs [hunk 1]
```diff
@@ -8,33 +8,11 @@ namespace Microsoft.Recognizers.Text.Number.Spanish
...
-                var instance = new NumberExtractor(mode, options);
-                Instances.TryAdd(cacheKey, instance);
-            }
-
-            return Instances[cacheKey];
-        }
-
         private NumberExtractor(NumberMode mode = NumberMode.Default, NumberOptions options = NumberOptions.None)
         {
-            NegativeNumberTermsRegex = new Regex(NumbersDefinitions.NegativeNumberTermsRegex + '$', RegexOptions.Singleline);
```
Comment (JuanAr) [inline .NET/Microsoft.Recognizers.Text.Number/Spanish/Extractors/NumberExtractor.cs]: Where did this line went? Is it part of a fix or just a typo ([DELETE] typo)

**Review by JuanAr — CHANGES_REQUESTED** [review]
There seems to be a missing line in the code. Also, the build failed for this specific commit.

File: .NET/Microsoft.Recognizers.Text.Number/Spanish/Extractors/NumberExtractor.cs (hunk 1 above)
Comment (Aliandi) [reply .NET/Microsoft.Recognizers.Text.Number/Spanish/Extractors/NumberExtractor.cs]: Solved, it was a typo. Changes pushed.

**Review by ceciliaavila — APPROVED** [review]
LGTM

**Review by JuanAr — APPROVED** [review]

**Pull request closed without merging**